service/tasks/monitoring.py

```diff
@@ -520,10 +511,66 @@ def monitor_instances_for(provider_id, users=None,
                 provider, username,
                 print_logs, start_date, end_date)
     if print_logs:
-        celery_logger.removeHandler(consolehandler)
+        _exit_stdout_logging(console_handler)
     return running_total
 
 
+@task(name="monitor_volumes")
+def monitor_volumes():
+    """
+    Update volumes for each active provider.
+    """
+    for p in Provider.get_active():
+        monitor_volumes_for.apply_async(args=[p.id])
+
+@task(name="monitor_volumes_for")
+def monitor_volumes_for(provider_id, print_logs=False):
```

Comment:
This is the code up for review. Apparently I leaked some doc-string changes into this branch :)